backend/src/main/scala/cromwell/backend/async/AsyncBackendJobExecutionActor.scala

```diff
@@ -35,7 +36,15 @@ trait AsyncBackendJobExecutionActor { this: Actor with ActorLogging =>
 
   def retryable: Boolean
 
-  def isFatal(throwable: Throwable): Boolean = throwable.isInstanceOf[CromwellFatalExceptionMarker]
+  // The scala package object (scala/package.scala) contains a neat list of runtime errors that are always going to be fatal.
```

Comment:
ToL: Chris- I think believe your referring to the aliases of errors / exception here: https://github.com/scala/scala/blob/v2.11.8/src/library/scala/package.scala#L15-L30

For the purposes of ToL discussion, there's also scala's `NonFatal(Throwable)` extractor, that I don't like btw, because it treats some `Throwable`s as "ok": https://github.com/scala/scala/blob/v2.11.8/src/library/scala/util/control/NonFatal.scala